src/element/newElement.ts

```diff
@@ -46,28 +53,37 @@ type ElementConstructorOpts = MarkOptional<
   | "version"
   | "versionNonce"
   | "link"
+  | "strokeStyle"
+  | "fillStyle"
+  | "strokeColor"
+  | "backgroundColor"
+  | "roughness"
+  | "strokeWidth"
+  | "roundness"
+  | "locked"
+  | "opacity"
 >;
 
 const _newElementBase = <T extends ExcalidrawElement>(
   type: T["type"],
   {
     x,
     y,
-    strokeColor,
-    backgroundColor,
-    fillStyle,
-    strokeWidth,
-    strokeStyle,
-    roughness,
-    opacity,
+    strokeColor = oc.black,
+    backgroundColor = "transparent",
+    fillStyle = "hachure",
+    strokeWidth = 1,
+    strokeStyle = "solid",
+    roughness = 1,
+    opacity = 100,
```

Comment:
we should reuse `getDefaultAppState()` values, or factor those values from the function return value into constants similarly as we do with `DEFAULT_TEXT_ALIGN`

Reply:
Done, exported constant for these values and we can keep adding more, probably font size, textalign could also move into the `DEFAULT_ELEMENT_PROPS`